Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java

```diff
@@ -584,8 +584,12 @@ public void onViewHolderBind(VPAdapter.ViewHolder holder, int position, Context
                 
     @JavascriptInterface
     public void pauseYoutubeVideo() {
-        if (youTubePlayerView != null) {
-            youtubePlayer.pause();
+        if(vpAdapter != null){
+            vpAdapter.pauseYoutubeVideo();
+            if (youTubePlayerView != null) {
+                if(youtubePlayer != null)
+                    youtubePlayer.pause();
```

Comment:
This condition can be combined here  `(youTubePlayerView != null && youtubePlayer != null)`